torch/csrc/onnx/init.cpp

```diff
@@ -39,7 +39,7 @@ void initONNXBindings(PyObject* module) {
 
   onnx.attr("PRODUCER_VERSION") = py::str(TORCH_VERSION);
 
-#ifdef PYTORCH_ONNX_CAFFE2_BUNDLE
+#ifdef BUILD_CAFFE2
```

Comment:
I'm OK with relying on this variable assuming that it defaults to false in the next PyTorch release. I believe that's the current plan, but we'll need to have a back-up plan ready to go in case they decide to keep it defaulting to On.

https://github.com/pytorch/pytorch/blob/76efbccc3b970c391ad147f29af960a6f028014e/CMakeLists.txt#L151

I guess the back-up plan could be to add a new export type.

Reply:
As of https://github.com/pytorch/pytorch/pull/66658 (merged on Oct 21st) Caffe2 is not built by default with PyTorch anymore.

If that changes at some point, which is really unlikely as Caffe2 is being put to sleep, we can create a new operator export type as the original PR proposed.